e2e/exit-exam-modal.spec.ts

```diff
@@ -0,0 +1,97 @@
+import { test, expect, type Page } from '@playwright/test';
+import translations from '../client/i18n/locales/english/translations.json';
+
+test.use({ storageState: 'playwright/.auth/certified-user.json' });
+
+let page: Page;
+const examUrl =
+  '/learn/foundational-c-sharp-with-microsoft/foundational-c-sharp-with-microsoft-certification-exam/foundational-c-sharp-with-microsoft-certification-exam';
+const cancelExamUrl =
+  '/learn/foundational-c-sharp-with-microsoft/#foundational-c-sharp-with-microsoft-certification-exam';
+
+test.beforeAll(async ({ browser }) => {
+  page = await browser.newPage();
+});
+
+test.afterAll(async () => {
+  await page.close();
+});
```

Comment:
```suggestion
test.afterAll(async ({ page }) => {
  await page.close();
});
```

We should run the tests in isolation, ideally.

Since the before hook was just being used to instantiate a page, we don't need it.